transport/transport-runtime/src/main/java/com/google/android/datatransport/runtime/ForcedSender.java

```diff
@@ -0,0 +1,26 @@
+package com.google.android.datatransport.runtime;
+
+import android.annotation.SuppressLint;
+import androidx.annotation.Discouraged;
+import androidx.annotation.WorkerThread;
+import com.google.android.datatransport.Priority;
+import com.google.android.datatransport.Transport;
+
+@Discouraged(
+    message =
+        "TransportRuntime is not a realtime delivery system, don't use unless you absolutely must.")
+public final class ForcedSender {
+  @WorkerThread
+  public static void sendBlockingWithPriority(Transport<?> transport, Priority priority) {
```

Comment:
nit 
```suggestion
  public static void sendBlocking(Transport<?> transport, Priority priority) {
```

arguments are part of the method signature, so no need to include the priority in the method name